### ISSUE TITLE
Owner can see all text channels at all times

### ISSUE THREAD

**Issue by Karstenan** [GLedoussal/discord-vocaltext#3]
I am rather new to discord so I do not know if there is an easy fix to this problem.
I connected your bot to a team and setup matching text channels to each voice channel, made a group specifically for the bot and gave it the permissions it needs to add and remove people from the text channels. As well as making the bot a member of each text channel.

This works well for all users without the General Permissions "Administrator", the problem is that the owner account is at all times a member in all the text channels, so the adding removing does not work for the owner account.

Is there an easy fix to this that can either be done by playing around with the permissions or programmatically in the script?

I can try to implement it myself if you point me in the right direction as my javascript and discord experience is currently lacking.

**Comment by khalen56** [comment]
I can't fix this, because admin permission is above all other permissions.

The only thing you can do is creating a secondary account and transfer the owners rights. That's what I did on my server.

**Comment by Karstenan** [comment]
Yeah I was  afraid of that being the only solution. Ah well, with that exception it seems to work quite well. Good job :)
